test/v2/controllers/V2ArrivalsControllerSpec.scala

```diff
@@ -675,4 +682,95 @@ class V2ArrivalsControllerSpec
     }
   }
 
+  "GET  /traders/:EORI/movements/arrivals" - {
+    "should return ok with json body for arrivals" in {
+
+      val enrolmentEORINumber = arbitrary[EORINumber].sample.value
+      val dateTime            = OffsetDateTime.of(2022, 8, 4, 11, 34, 42, 0, ZoneOffset.UTC)
+
+      val movementResponse1 = MovementResponse(
+        _id = arbitrary[MovementId].sample.value,
+        enrollmentEORINumber = enrolmentEORINumber,
+        movementEORINumber = arbitrary[EORINumber].sample.value,
```

Comment:
can use the Seq of Arbitrary[MovementResponse] instead of creating two new instances here. Not sure.
@liandra-giacomine / @drnaylor-oc any suggestions

Reply:
Could use arbitraryMovementResponse from CommonGenerators, then use the .copy method to just update the relevant parameters